src/modules/add-score.js

```diff
@@ -0,0 +1,11 @@
+const addApiScore = (user) => {
+  fetch('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/np8G7JdKLR96QcXBTm5K/scores/', {
```

Comment:
- [ ] As per the previous reviewer,  Kindly use async and await in all the API calls in your code as mentioned in the project requirements. KIndly implement it.